src/freeze.tsx

```diff
@@ -79,6 +79,18 @@ function updateMetadata(state: CellState, cell: Cell<ICellModel>) {
   cell.update();
 }
 
+export function changeState(state: CellState, notebookPanel: NotebookPanel) {
+  const notebook = notebookPanel.content;
+  const currentCell = notebook.widgets[notebook.activeCellIndex];
+  if (currentCell) {
+    const currentState = getState(currentCell);
+    const newState: CellState = currentState === state ? 'normal' : state;
+    updateMetadata(newState, currentCell);
+    notebookPanel.update();
+  }
```

Comment:
Recently we changed this to allow multi-cell selection. Could you rewrite the code similar to what we have in `buttonCallback()` to allow multicell selection?

Reply:
Gotcha, edited!